### DISCUSSION TITLE
Supporter Tier Packs - Additional Options

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#598] [Suggestions]
As it currently stands, under the **Settings** of individual Supporter Tier packs, there are no options for frequency, role, character level, or server restrictions. At the very least, having restrictions on frequency of purchases would be beneficial for certain created tasks (such as implementing server boosts/weekend rates via config swaps). 

![image](https://github.com/gameserverapp/Platform/assets/44717221/d5159d02-2a78-449e-a186-49da03bddf5f)

![image](https://github.com/gameserverapp/Platform/assets/44717221/457b0ea2-c4e7-4912-89e6-0b55ee83ad96)

**Comment by zerafox** [comment]
Not exactly sure I understood what this means, but if it's referring to being able to setup a "Schedule" system based on config changes so they aren't cancelling eachother out or whatever, I'm all for it!

We'd like to be able to offer boosts to certain functions on our server but are afraid that if we offer one package say x2 Taming and another one to do x2 Gathering. The one purchased after would then swap to another config and cancel out the first.

Not sure the best way to handle this, but it does make any changes to rates or settings a non-viable option in a variety of ways simply because they'd cancel one another out in some way or another.

**Comment by InfinityofphoenixTimo** [comment]
Personally, I really miss the option in the Support Tier Shop to choose how many times a user can purchase a package. I constantly have to process refunds because non-stackable memberships are purchased twice. Also, things like battle passes, memberships, monthly access, etc., which work with plugins or other integrations, would be much easier to manage. We've often had problems with this and would like to have the features of the token shop in the Supporter Shop as well. At least the limits so that double purchases of single products aren't possible.
